### ISSUE TITLE
trendline does not support different line style?

### ISSUE THREAD

**Issue by reinertisa** [Makanz/chartjs-plugin-trendline#81]
Hi, 
<img width="871" alt="Screenshot 2023-08-21 at 11 41 17 AM" src="https://github.com/Makanz/chartjs-plugin-trendline/assets/71358643/6f6da9dd-5431-4751-a10b-d29ff7548e4a">
I use two datasets and display two different trendline. I expect one is dotted and another one is solid. But of them are dotted. Can you check it out when you can, please? @Makanz 

```
[
    {
        "label": "Dataset 1",
        "data": [
            10,
            30,
            50,
            20,
            25,
            44,
            -10
        ],
        "borderColor": "#4dc9f6",
        "backgroundColor": "#4dc9f6",
        "trendlineLinear": {
            "colorMin": "black",
            "colorMax": "black",
            "lineStyle": "dotted",
            "width": 2,
            "projection": false
        }
    },
    {
        "label": "Dataset 2",
        "data": [
            100,
            33,
            22,
            19,
            11,
            49,
            200
        ],
        "borderColor": "#f67019",
        "backgroundColor": "#f67019",
        "trendlineLinear": {
            "colorMin": "red",
            "colorMax": "red",
            "lineStyle": "solid",
            "width": 2,
            "projection": false
        }
    }
]

**Comment by Makanz** [comment]
I will take a look at it when possible.

**Comment by reinertisa** [comment]
I noticed if first dataset's lineStyle: solid and second dataset's lineStyle: dotted, it works. In our case this is opposite. I hope it helps you to find the solution. @Makanz

**Comment by Makanz** [comment]
@reinertisa I solved the bug in the latest version v2.0.5
https://github.com/Makanz/chartjs-plugin-trendline/releases/tag/v2.0.5

**Comment by reinertisa** [comment]
thanks, I tested and it works.
